diff --git a/ratebeer/models.py b/ratebeer/models.py
index 534c670..6f486d8 100644
--- a/ratebeer/models.py
+++ b/ratebeer/models.py
@@ -202,6 +195,29 @@ class Beer(object):
 
         return self
 
+    def get_review_comments(self):
+        if not self.id:
+            self.id = self.url.split('/')[-2]
+
+        data = [{"operationName": "beerReviews",
+                 "variables": {"beerId": self.id},
+                 "query": "query beerReviews($beerId: ID!) {\n beerReviews(beerId: $beerId, order: TOP_RATERS, first: 5000) {\n reviews: items {\n score\n comment\n createdAt\n author {\n username\n city\n state {\n name\n __typename\n }\n country {\n name\n __typename\n }\n __typename\n }\n __typename\n }\n __typename\n }\n}\n"
+                 }]
+
+        request = requests.post(
+            "https://beta.ratebeer.com/v1/api/graphql/",
+            data=json.dumps(data),
+            headers={"content-type": "application/json"}
+        )
+
+        try:
+            results = json.loads(request.text)
+        except:
+            raise rb_exceptions.JSONParseException(self.id)
+
+        for r in results[0]['data']['beerReviews']['reviews']:
+            yield r['comment']
+
     def get_reviews(self, review_order="most recent"):
         """Returns reviews for a specific beer.
 
